run.ps1

```diff
@@ -1,20 +1,30 @@
 Param([bool]$Manual = $false)
 
 function Get-Tags {
-    Param([string]$System, [string]$Variant)
+    Param([string]$System, [string]$Variant, [string]$Synergy)
     
-    return @($System, $Variant, "$System-$Variant")
+    return @($System, $Variant, $Synergy, "$System-$Variant-$Synergy")
 }
-function Get-TagsFromScannerInfo {
-    Param([string]$File)
+
+function Get-TagsFromConfigFiles {
+    Param([string]$File, [string]$SynergyPath)
 
     $info = Get-Content $File | Select-Object -Skip 3 | ConvertFrom-StringData
+    
+    
+    
+   if (Test-Path 'C:/ProgramData/coDiagnostiX/DWSynergySrv'){
+        $synergy = "Synergy"
+   } else {
+        $synergy = "None"
+   }

```

Comment:
I'd rather not add a tag "None", simply let $synergy be null and if null, not add it to the list returned

Reply:
so you would have 2 switch?

Reply:
No, I would simply do:
```
If (Test-Path 'C:/ProgramData/coDiagnostiX/DWSynergySrv'){
        $tags += "Synergy"
}
```
after the switch.
(And inline the `Get-Tags` method, which is over engineering things a bit)